IntegrationTests/src/bkr/inttest/server/test_kickstart.py

```diff
@@ -2322,12 +2324,13 @@ def test_custom_kickstart_centos7_with_conflicts_groups(self):
         self.assertIn('''\
 install
 %packages
+# Task requirements will be installed by the harness
 *
 -@conflicts-client
 -@conflicts-server
 -@conflicts-workstation
 %end
-''', ks)
+''', ks, ks)
```

Comment:
Why double ks?

Reply:
I added it there so you can see the whole output of KS in a formatted way.
Basically 1:1 to what we sending to boxes. I was dealing with several issues regarding this and you can't tell it from exception - Because it will screw output.